ibm/mas_devops/roles/kmodels/templates/kmodels-store/network.yml.j2

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: km-store
+  namespace: mas-{{ mas_instance_id }}-aibroker
+spec:
+  ports:
+    - name: http
```

Comment:
Hi @whitfiea 
There is no sensitive data in store so there is no need to use secure port - we will create work item to fix this in future

Reply:
@karol-czarnecki so there is no authentication that occurs and so anyone can call the route for any of these apis? That doesn't seem right to me and all connections should be secure in one way or another.

Reply:
The operator should be creating these services anyway, not the ansible code, whose job should just be to install the operator and create the operand.  The whole thing needs a rewrite frankly because it doesn't meet the MAS application contract (if it did we wouldn't have this special install role and would be able to re-use the standard mas_app_install, mas_app_cfg roles), but we're shipping it as-is anyway.